### PR TITLE
Feature/coverage

### LINKED ISSUE #33: Add coverage percentage to api client
Coverage percentage was implemented as of 2020-10-05 as an additional column on roi time series. This column contains the information on which area fraction contributed to the mean value. To request this, an additional keyword needs to be added to the time series requests: `provide_coverage=None` (default setting: `False`)

### PR DESCRIPTION
**Pull request by teije01**
This adds the coverage option to roi-time-series

closes #33

### CODE REVIEW

File: vds_api_client/api_cli.py [hunk 1]
```diff
@@ -286,6 +288,7 @@ def ts(ctx, config_file, products, latlons, rois, date_range, fmt,
                                  end_time=date_range[1] if date_range else None,
                                  lats=lats, lons=lons, rois=rois,
                                  av_win=av_win, av_win_dir=av_win_dir, masked=masked, clim=clim, t=t,
+                                 provide_coverage=coverage,
```
Comment (ajgeers) [inline vds_api_client/api_cli.py]: For consistency with other arguments, I prefer `coverage=coverage`.

Comment (teije01) [reply vds_api_client/api_cli.py]: then it should be `provide_coverage=provide_coverage` to be consisent with the api I guess. do you then also prefer `--provide_coverage`? It shortened it for less typing in the cli

Comment (teije01) [reply vds_api_client/api_cli.py]: It's now all `provide_coverage`

Comment (ajgeers) [reply vds_api_client/api_cli.py]: I see what you mean: `clim` and `t` are also shortened versions of the corresponding API parameters. If you prefer short, then `cov=cov` or `coverage=coverage` would also work for me.

Comment (teije01) [reply vds_api_client/api_cli.py]: Yes, to harmonize those would be a different issue (and a new major version). For now what I will do is keep the `provide_coverage` intact and add a shortcut `-cov` to `--provide-coverage` so both options are available.

Comment (ajgeers) [reply vds_api_client/api_cli.py]: Great, that works

**Review by ajgeers — APPROVED** [review]

**Comment by teije01** [comment]
You can merge if you've finished your review

**Pull request merged**